docs/index-all.html

```diff
@@ -1705,23 +1705,23 @@ <h2 class="title">C</h2>
 <dd>
 <div class="block">
  The metadata passing to output binding components
- 
+
```

Comment:
Please remove all doc changes, they will be generated and added when the SDK is released. only javadoc comments are necessary so the htlm docs can be generated.

Reply:
okay